source/reference/oplog-sizing.txt

```diff
@@ -63,6 +63,12 @@ Monitor oplog Size Needed for Initial Sync
       - Increase the ``oplog`` window. Use :dbcommand:`replSetResizeOplog`
         to set ``minRetentionHours`` greater than the current ``oplog`` 
         window.
+
+        .. note:: 
```

Comment:
Do c2c docs use a replacement for Atlas, like |service|? If so, use it here too. If not, then leave Atlas.

Reply:
Please review with the dev team as well, cc Ankit Luthra (via slack).